### DISCUSSION TITLE
Redeliver Option

### DISCUSSION THREAD

**Discussion by miroslavskela** [postalserver/postal#1657] [Help with using Postal] [answered]
Hello,
i have wondering how does redeliver option works, i tried to use it on some mails and postal says that mail is sent and accepted by some server, but recipient does not receive message again. One message from picture is not in inbox. Is that even a purpose of redeliver option, maybe this does not work if first message is sent ok? 
Also, is there a chance that postal says that message is sent, and received by server, but recipient never gets the message, it never ends in his mail inbox or junk.
![Screenshot from 2021-10-23 10-56-02](https://user-images.githubusercontent.com/36046629/138550176-f15c749e-037c-4e7b-9174-68ad38e71fc1.png)

**Comment by willpower232** [comment, marked as answer]
Its possible that because it was a successful delivery, gmail saw it already had that message and didn't bother duplicating it for the user, it looks like microsoft doesn't duplicate messages either.

IIRC redelivery is handy if a message has hard failed and won't be retried but you know the problem has been resolved.
